README.md

```diff
@@ -1,54 +1,89 @@
-# Try Cloaked Search
+## Try Cloaked Search (in ~5 Minutes)
 
-Please see the getting started steps and explanations at https://ironcorelabs.com/docs/saas-shield/cloaked-search/try-cloaked-search/
+Cloaked Search is a proxy for Elasticsearch that protects the indexed data from prying eyes. Cloaked Search's API is the same as the underlying Elasticsearch API.
 
-## Quick Start for the Impatient
+In about 5 minutes, you will have:
+* Elasticsearch running on your local machine
+* Cloaked Search running on your local machine
+* sample data indexed with `title` as a protected field
+* query results from sample queries using the protected `title` field
 
-Get things started by running the following two commands from this repo's directory but in separate terminal windows:
+All stored data is on a temporary volume inside of docker. No changes will be made your machine beyond `try-cloaked-search` repo.
 
+## Dependencies
+
+To try Cloaked Search you just need a basic *nix installation and `docker` + `docker-compose`. Some of the commands below also use `jq` for JSON formatting. If you don't have `jq`, you can safely remove those portions of the command.
+
+## Get Cloaked Search Running
+
+Clone the [try-cloaked-search](https://github.com/IronCoreLabs/try-cloaked-search) git repo.
+
+```bash
+git clone https://github.com/IronCoreLabs/try-cloaked-search.git
 ```
+
+All other commands are assumed be run from within the cloned repo.
+
+### Start Cloaked Search and Elasticsearch
+
+```bash
 docker-compose up
+```
+
+**Note: Future commands will be targeting Cloaked Search on port 8675**
+
+## Indexing
+
+`try-cloaked-search` includes some test data. Since Cloaked Search uses a different key per (tenant, index, field), documents with protected fields must be tagged with the tenant they belong to. Half of the articles in the test dataset are associated with `tenant-1`, and the other half are not associated with any tenant. Only documents belonging to `tenant-1` will be encrypted. To better understand Cloaked Search's key management, refer to the [configuration documentation](/docs/saas-shield/cloaked-search/configuration).
+
+```bash
 ./populate_index.sh
 ```
 
-Then play with the shell scripts to see what happens with commands like this:
+### (optional) Look at an encrypted index
 
+_Note that we are making this request directly to Elasticsearch (port 9200) so we can see what's actually stored._
+
+Let's get all the documents belonging to `tenant-1` and see what's in the index!
+
+```bash
+curl -s -G --data-urlencode "q=tenant_id:\"tenant-1\"" localhost:9200/try_cloaked_search/_search | jq
```

Comment:
Should this query (and all our other sample queries) use the `+tenant_id` like we're going to require?

Reply:
yep. added.